src/System.Runtime.Extensions/ref/project.json

```diff
@@ -1,12 +1,9 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.20"
+    "System.Runtime": "4.1.0"
```

Comment:
Can we change this to the latest System.Runtime package prerelease? that way you will get netstandard1.7 version of System.Runtime surface area.